<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MM</class>
 <widget class="QMainWindow" name="MM">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1225</width>
    <height>668</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Aker System - Mission Monitor</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <property name="styleSheet">
    <string notr="true"/>
   </property>
   <layout class="QGridLayout" name="gridLayout_2" rowstretch="0,0,0,0">
    <property name="leftMargin">
     <number>1</number>
    </property>
    <property name="topMargin">
     <number>0</number>
    </property>
    <property name="rightMargin">
     <number>1</number>
    </property>
    <property name="bottomMargin">
     <number>0</number>
    </property>
    <property name="spacing">
     <number>0</number>
    </property>
    <item row="0" column="5">
     <widget class="QFrame" name="frame_2">
      <property name="sizePolicy">
       <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
        <horstretch>1</horstretch>
        <verstretch>0</verstretch>
       </sizepolicy>
      </property>
      <property name="minimumSize">
       <size>
        <width>380</width>
        <height>256</height>
       </size>
      </property>
      <property name="styleSheet">
       <string notr="true">image: url(:/resources/Images/Logo.png);
border:0px;</string>
      </property>
      <property name="frameShape">
       <enum>QFrame::StyledPanel</enum>
      </property>
      <property name="frameShadow">
       <enum>QFrame::Raised</enum>
      </property>
      <zorder>horizontalSpacer_2</zorder>
      <zorder>horizontalSpacer</zorder>
     </widget>
    </item>
    <item row="0" column="7">
     <spacer name="horizontalSpacer_2">
      <property name="orientation">
       <enum>Qt::Horizontal</enum>
      </property>
      <property name="sizeHint" stdset="0">
       <size>
        <width>300</width>
        <height>20</height>
       </size>
      </property>
     </spacer>
    </item>
    <item row="0" column="3">
     <spacer name="horizontalSpacer">
      <property name="orientation">
       <enum>Qt::Horizontal</enum>
      </property>
      <property name="sizeHint" stdset="0">
       <size>
        <width>300</width>
        <height>20</height>
       </size>
      </property>
     </spacer>
    </item>
    <item row="2" column="3" colspan="5">
     <layout class="QHBoxLayout" name="horizontalLayout">
      <property name="spacing">
       <number>0</number>
      </property>
      <item>
       <layout class="QGridLayout" name="gridLayout">
        <property name="spacing">
         <number>0</number>
        </property>
        <item row="5" column="1">
         <widget class="QLCDNumber" name="val2">
          <property name="styleSheet">
           <string notr="true">border:1px solid rgba(255, 255, 255, 50);
color:rgba(255, 0, 0, 128);

</string>
          </property>
          <property name="value" stdset="0">
           <double>-1.000000000000000</double>
          </property>
         </widget>
        </item>
        <item row="4" column="0">
         <widget class="QLabel" name="label_1">
          <property name="sizePolicy">
           <sizepolicy hsizetype="Expanding" vsizetype="Expanding">
            <horstretch>1</horstretch>
            <verstretch>1</verstretch>
           </sizepolicy>
          </property>
          <property name="styleSheet">
           <string notr="true">border:1px solid rgba(255, 255, 255, 50);
color:rgba(255, 255, 255, 220);
font: 36pt &quot;Ubuntu Condensed&quot;;</string>
          </property>
          <property name="text">
           <string>Tens達o Charger</string>
          </property>
         </widget>
        </item>
        <item row="4" column="1">
         <widget class="QLCDNumber" name="val1">
          <property name="sizePolicy">
           <sizepolicy hsizetype="Expanding" vsizetype="Expanding">
            <horstretch>1</horstretch>
            <verstretch>1</verstretch>
           </sizepolicy>
          </property>
          <property name="styleSheet">
           <string notr="true">border:1px solid rgba(255, 255, 255, 50);
color:rgba(255, 0, 0, 128);
</string>
          </property>
          <property name="digitCount">
           <number>5</number>
          </property>
          <property name="segmentStyle">
           <enum>QLCDNumber::Filled</enum>
          </property>
          <property name="value" stdset="0">
           <double>-1.000000000000000</double>
          </property>
         </widget>
        </item>
        <item row="2" column="0">
         <widget class="QLabel" name="label_4">
          <property name="sizePolicy">
           <sizepolicy hsizetype="Expanding" vsizetype="Expanding">
            <horstretch>1</horstretch>
            <verstretch>1</verstretch>
           </sizepolicy>
          </property>
          <property name="styleSheet">
           <string notr="true">border:1px solid rgba(255, 255, 255, 50);
color:rgba(255, 255, 255, 220);
font: 36pt &quot;Ubuntu Condensed&quot;;</string>
          </property>
          <property name="lineWidth">
           <number>1</number>
          </property>
          <property name="text">
           <string>Tens達o Bateria</string>
          </property>
          <property name="textFormat">
           <enum>Qt::AutoText</enum>
          </property>
         </widget>
        </item>
        <item row="6" column="0">
         <widget class="QLabel" name="label">
          <property name="sizePolicy">
           <sizepolicy hsizetype="Expanding" vsizetype="Expanding">
            <horstretch>1</horstretch>
            <verstretch>1</verstretch>
           </sizepolicy>
          </property>
          <property name="styleSheet">
           <string notr="true">border:1px solid rgba(255, 255, 255, 50);
color:rgba(255, 255, 255, 220);
font: 36pt &quot;Ubuntu Condensed&quot;;
</string>
          </property>
          <property name="text">
           <string>Tens達o Painel 2</string>
          </property>
          <property name="scaledContents">
           <bool>true</bool>
          </property>
          <property name="alignment">
           <set>Qt::AlignLeading|Qt::AlignLeft|Qt::AlignVCenter</set>
          </property>
         </widget>
        </item>
        <item row="5" column="0">
         <widget class="QLabel" name="label_2">
          <property name="sizePolicy">
           <sizepolicy hsizetype="Expanding" vsizetype="Expanding">
            <horstretch>0</horstretch>
            <verstretch>1</verstretch>
           </sizepolicy>
          </property>
          <property name="styleSheet">
           <string notr="true">border:1px solid rgba(255, 255, 255, 50);
color:rgba(255, 255, 255, 220);
font: 36pt &quot;Ubuntu Condensed&quot;;</string>
          </property>
          <property name="text">
           <string>Tens達o Painel 1</string>
          </property>
         </widget>
        </item>
        <item row="6" column="1">
         <widget class="QLCDNumber" name="val3">
          <property name="styleSheet">
           <string notr="true">border:1px solid rgba(255, 255, 255, 50);
color:rgba(255, 0, 0, 128);
</string>
          </property>
          <property name="value" stdset="0">
           <double>-1.000000000000000</double>
          </property>
         </widget>
        </item>
        <item row="2" column="1">
         <widget class="QLCDNumber" name="val0">
          <property name="sizePolicy">
           <sizepolicy hsizetype="Expanding" vsizetype="Expanding">
            <horstretch>1</horstretch>
            <verstretch>1</verstretch>
           </sizepolicy>
          </property>
          <property name="minimumSize">
           <size>
            <width>0</width>
            <height>0</height>
           </size>
          </property>
          <property name="styleSheet">
           <string notr="true">border:1px solid rgba(255, 255, 255, 50);
color:rgba(255, 0, 0, 128);
</string>
          </property>
          <property name="smallDecimalPoint">
           <bool>false</bool>
          </property>
          <property name="digitCount">
           <number>5</number>
          </property>
          <property name="value" stdset="0">
           <double>-1.000000000000000</double>
          </property>
         </widget>
        </item>
       </layout>
      </item>
      <item>
       <spacer name="horizontalSpacer_3">
        <property name="orientation">
         <enum>Qt::Horizontal</enum>
        </property>
        <property name="sizeType">
         <enum>QSizePolicy::Minimum</enum>
        </property>
        <property name="sizeHint" stdset="0">
         <size>
          <width>225</width>
          <height>20</height>
         </size>
        </property>
       </spacer>
      </item>
      <item>
       <layout class="QVBoxLayout" name="verticalLayout_2">
        <property name="spacing">
         <number>0</number>
        </property>
        <item>
         <widget class="QLabel" name="label_3">
          <property name="sizePolicy">
           <sizepolicy hsizetype="Expanding" vsizetype="Minimum">
            <horstretch>1</horstretch>
            <verstretch>0</verstretch>
           </sizepolicy>
          </property>
          <property name="minimumSize">
           <size>
            <width>0</width>
            <height>50</height>
           </size>
          </property>
          <property name="styleSheet">
           <string notr="true">border:1px solid rgba(255, 255, 255, 50);
color:rgba(255, 255, 255, 220);
font: 42pt &quot;Ubuntu Condensed&quot;;</string>
          </property>
          <property name="text">
           <string>Comandos</string>
          </property>
          <property name="alignment">
           <set>Qt::AlignCenter</set>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QFrame" name="comandos">
          <property name="sizePolicy">
           <sizepolicy hsizetype="Expanding" vsizetype="Expanding">
            <horstretch>0</horstretch>
            <verstretch>0</verstretch>
           </sizepolicy>
          </property>
          <property name="styleSheet">
           <string notr="true">border:1px solid rgba(255, 255, 255, 50);
color:rgba(255, 255, 255, 220);
font: 36pt &quot;Ubuntu Condensed&quot;;</string>
          </property>
          <property name="frameShape">
           <enum>QFrame::StyledPanel</enum>
          </property>
          <property name="frameShadow">
           <enum>QFrame::Raised</enum>
          </property>
          <layout class="QVBoxLayout" name="verticalLayout">
           <property name="spacing">
            <number>0</number>
           </property>
           <property name="leftMargin">
            <number>0</number>
           </property>
           <property name="topMargin">
            <number>0</number>
           </property>
           <property name="rightMargin">
            <number>0</number>
           </property>
           <property name="bottomMargin">
            <number>0</number>
           </property>
           <item>
            <widget class="QRadioButton" name="radio_0">
             <property name="sizePolicy">
              <sizepolicy hsizetype="Minimum" vsizetype="Expanding">
               <horstretch>0</horstretch>
               <verstretch>0</verstretch>
              </sizepolicy>
             </property>
             <property name="styleSheet">
              <string notr="true">color:rgba(255, 255, 255, 220);
font: 32pt &quot;Ubuntu Condensed&quot;;</string>
             </property>
             <property name="text">
              <string>Lan巽ar Antena</string>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QRadioButton" name="radio_1">
             <property name="sizePolicy">
              <sizepolicy hsizetype="Minimum" vsizetype="Expanding">
               <horstretch>0</horstretch>
               <verstretch>0</verstretch>
              </sizepolicy>
             </property>
             <property name="styleSheet">
              <string notr="true">color:rgba(255, 255, 255, 220);
font: 32pt &quot;Ubuntu Condensed&quot;;</string>
             </property>
             <property name="text">
              <string>Determina巽達o de Atitude</string>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QRadioButton" name="radio_2">
             <property name="sizePolicy">
              <sizepolicy hsizetype="Minimum" vsizetype="Expanding">
               <horstretch>0</horstretch>
               <verstretch>0</verstretch>
              </sizepolicy>
             </property>
             <property name="styleSheet">
              <string notr="true">color:rgba(255, 255, 255, 220);
font: 32pt &quot;Ubuntu Condensed&quot;;</string>
             </property>
             <property name="text">
              <string>Estabiliza巽達o</string>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QRadioButton" name="radio_3">
             <property name="sizePolicy">
              <sizepolicy hsizetype="Expanding" vsizetype="Expanding">
               <horstretch>0</horstretch>
               <verstretch>0</verstretch>
              </sizepolicy>
             </property>
             <property name="text">
              <string>Apontamento</string>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QRadioButton" name="radio_4">
             <property name="sizePolicy">
              <sizepolicy hsizetype="Minimum" vsizetype="Expanding">
               <horstretch>0</horstretch>
               <verstretch>0</verstretch>
              </sizepolicy>
             </property>
             <property name="styleSheet">
              <string notr="true">color:rgba(255, 255, 255, 220);
font: 32pt &quot;Ubuntu Condensed&quot;;</string>
             </property>
             <property name="text">
              <string>Ler Par但metros El辿tricos</string>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QRadioButton" name="radio_5">
             <property name="sizePolicy">
              <sizepolicy hsizetype="Minimum" vsizetype="Expanding">
               <horstretch>0</horstretch>
               <verstretch>0</verstretch>
              </sizepolicy>
             </property>
             <property name="styleSheet">
              <string notr="true">color:rgba(255, 255, 255, 220);
font: 32pt &quot;Ubuntu Condensed&quot;;</string>
             </property>
             <property name="text">
              <string>Imageamento</string>
             </property>
            </widget>
           </item>
          </layout>
         </widget>
        </item>
        <item>
         <widget class="QPushButton" name="sendComm">
          <property name="sizePolicy">
           <sizepolicy hsizetype="Minimum" vsizetype="Minimum">
            <horstretch>0</horstretch>
            <verstretch>0</verstretch>
           </sizepolicy>
          </property>
          <property name="minimumSize">
           <size>
            <width>0</width>
            <height>42</height>
           </size>
          </property>
          <property name="styleSheet">
           <string notr="true">color:rgba(255, 255, 255, 220);
font: 32pt &quot;Ubuntu Condensed&quot;;</string>
          </property>
          <property name="text">
           <string>Send</string>
          </property>
         </widget>
        </item>
       </layout>
      </item>
     </layout>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>1225</width>
     <height>20</height>
    </rect>
   </property>
   <widget class="QMenu" name="menuBack">
    <property name="styleSheet">
     <string notr="true">color:rgb(255, 255, 255);</string>
    </property>
    <property name="title">
     <string>Help</string>
    </property>
    <addaction name="actionBack"/>
   </widget>
   <addaction name="menuBack"/>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
  <action name="actionBack">
   <property name="text">
    <string>Back</string>
   </property>
  </action>
 </widget>
 <resources/>
 <connections/>
</ui>
